README.md

```diff
@@ -1,5 +1,30 @@
 # Analytic Hierarchy Process Solver in Python
 [![license](https://img.shields.io/github/license/mashape/apistatus.svg)]()
 
+## Theory
+Saaty's AHP model for making decision
+[AHP](https://en.wikipedia.org/wiki/Analytic_hierarchy_process)
```

Comment:
Change this citation to the original paper by Saaty instead of Wikipedia as it is a much more reliable source.